src/components/StaffList/StaffList.scss

```diff
@@ -0,0 +1,53 @@
+$light-gray: #aaa;
+$medium-gray: #ddd;
+
+.react-autosuggest__container {
+  position: relative;
+  margin-bottom: 300px;
+}
+
+.react-autosuggest__input {
+  width: 100%;
+  height: 30px;
+  padding: 10px 20px;
+  border: 1px solid $medium-gray;
+  border-radius: 0.1875rem;
+}
+
+.react-autosuggest__input--focused {
+  outline: none;
+}
+
+.react-autosuggest__input--open {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
```

Comment:
Are you running the edx stylelint against this? I think it would ask that this be updated to;
```
border: {
  bottom-left-radius: 0;
  bottom-right-radius: 0;
}
```

Reply:
i do whatever the pre-push thingy looks for

Reply:
how do i do stylelint

Reply:
- https://github.com/edx/prospectus/blob/781779314759e8f2db941e09398ec3ad0545d1e9/package.json#L88
- https://github.com/edx/prospectus/blob/781779314759e8f2db941e09398ec3ad0545d1e9/package.json#L99
- https://github.com/edx/prospectus/blob/cbc299376daf9145a450c6bb6e876dc9c1fa0022/.stylelintrc

should get you most of the way

Reply:
are you asking me to bring stylelint into PFE?